samps/contests/mini/config/contest.yaml

```diff
@@ -35,6 +35,10 @@ default-validator: /home/judge/cdp/config/validefault {:infile} {:outfile} {:ans
 # Load data files into pc2 as SerializedFiles, not using CDP external files
 load-data-files: false
 
+# load sample files then load secret data files, default is true
+# load-sample-judges-data : false
+# load-sample-judges-data : true
+

```

Comment:
I do not understand this change.  The text comment (3 lines up) says "default is true", but BOTH of the lines just above are commented-out.  Was the intent that the second line should have been uncommented (making the comment correct)?  Otherwise, I'm totally mystified as to why there are two lines (39 and 40), both comments, giving opposite values.

Reply:
Oh, unless the comment means that in the CODE the "default is true", and this sample YAML provides two options, both of which are currently commented-out but where the user can choose to uncomment one of them to override the code default?   That would make sense... if the comment explained it.  If that's the intent, I think the comment should be expanded to clarify the intent...